WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1113,13 +1054,12 @@ public void run() {
             QuickStartUtils.addQuickStartFocusPointAboveTheView(parentView, quickStartTarget, horizontalOffset,
                     verticalOffset);
 
-            // highlighting MySite row and scrolling to it
+            // highlight MySite row and scroll to it
             if (!QuickStartMySitePrompts.isTargetingBottomNavBar(mActiveTutorialPrompt.getTask())) {
                 mScrollView.post(new Runnable() {
-                    @Override
-                    public void run() {
-                        mScrollView.smoothScrollTo(0, quickStartTarget.getBottom());
-                        quickStartTarget.setPressed(true);
+                    @Override public void run() {
```

Comment:
```suggestion
                    @Override
                    public void run() {
```

Can we move the `@Override` annotation to a separate line than the method declaration for code style consistency?